UnityProject/Assets/Scripts/Player/PlayerNetworkActions.cs

```diff
@@ -660,6 +661,27 @@ public void CmdRequestPaperEdit(GameObject paper, string newMsg)
 		}
 	}
 
+	[Command]
+	public void CmdRequestRename(GameObject target, string customName)
+	{
+		var rename = target.GetComponent<Renameable>();
+
+		if (rename == null)
+		{
+			return;
+		}
+
+		if (customName.Length > 42)
+		{
+			customName = customName.Substring(0, 42);
+		}
+
+		customName = Regex.Replace(customName, "<size=\"(.*)\">", "");
```

Comment:
woops, bypassed :)
![image](https://user-images.githubusercontent.com/10403536/66701359-44b6e600-ed04-11e9-9948-9bd87c016873.png)
it has to be case insensitive

Reply:
Added case insensitive to the regex